core-bundle/src/Resources/contao/classes/Frontend.php

```diff
@@ -355,7 +355,7 @@ public static function getRootPageFromUrl()
 				// Redirect if the page alias is not "index" or "/" (see #8498, #8560 and #1210)
 				elseif (($objPage = \PageModel::findFirstPublishedByPid($objRootPage->id)) !== null && !\in_array($objPage->alias, array('index', '/')))
 				{
-					static::redirect($objPage->getFrontendUrl(), 302);
+					static::redirect($objPage->getFrontendUrl(), 307);
```

Comment:
Same here, since that is the new default value. Or do you always want to set the status explicitly?

Reply:
Removed in d66a47e47a557da17b7633d551b8f8038df78e92.